Formula/docker-machine-driver-hyperkit.rb

```diff
@@ -0,0 +1,35 @@
+class DockerMachineDriverHyperkit < Formula
+  desc "Docker Machine driver for hyperkit"
+  homepage "https://github.com/machine-drivers/docker-machine-driver-hyperkit"
+  url "https://github.com/machine-drivers/docker-machine-driver-hyperkit.git",
+    :tag => "v1.0.0",
```

Comment:
Align this and `:revision` with the first `"` above, thanks!

Reply:
Updated 👍